### DISCUSSION TITLE
Use a second account

### DISCUSSION THREAD

**Discussion by mirronake** [Heroic-Games-Launcher/HeroicGamesLauncher#882] [Ideas]
**I want to be able to log in and switch between Epic Games accounts.**

**Comment by flavioislima** [comment]
currently, we do not support this and its also no planned.

**Reply by mirronake** [reply]
Why? I'm sure I'm not the only one with many accounts.

**Reply by flavioislima** [reply]
Its not simple to implement it and maintain. will be a source for many bugs and corrupted files.
So will be a headache to maintain. 
This is a feature a lot of people asks but have you asked yourself how many launchers, even the ones with several and several millions of dollars involved like Steam, Ubisoft or even OSS like Lutris support this feature?

**Reply by mirronake** [reply]
And why not distinguish yourself and make a similar function, many people ask all launchers for a similar function.

**Reply by mirronake** [reply]
Maybe I'm wrong on some side, but this is a common suggestion to improve your project.

**Reply by noclue** [reply]
I am able to log out and log in to a second GoG account. This switches my library.

It would be great if Heroic can keep the credentials as typing passwords on the SteamDeck is slow.

It would be fantastic if accounts switch automatically when launching installed games. It seems I need to log to the proper account to launch a game.

My problem is that I have 2 GoG accounts by mistake and have Cyberpunk and Witcher 3 in the two accounts. I installed both and linked them to Steam. All works great except I need to type in password in Heroic to switch between the games. Games stick in starting state if I am logged into the wrong account.

**Reply by noclue** [reply]
I see the ask to preserve the password for more than one account has been made on few occasions in the past and now is rejected

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1782
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/1534

in the light of keyboard-less device like SteamDeck this becomes more important as re-typing the password is very  slow.
